packages/axios-extension/src/factory.ts

```diff
@@ -80,23 +86,53 @@ export function createForAbap(config: AxiosRequestConfig & Partial<ProviderConfi
     return createInstance(AbapServiceProvider, config);
 }
 
+export enum Authentication {
+    OAuth = 'OAuth',
+    ReentranceTicket = 'ReentranceTicket'
+}
+
+export interface AbapEnvironmentBtp {
```

Comment:
I don't like the interface names and both need a jsdoc explaining their different usages. I would prefer to rename them to describe the technical difference e.g. one using reentrance tickets the other oauth or to rename them based on their target e.g. standalone vs. embedded steampunk

Reply:
Will do. I prefer to use the target names (standalone, embedded Steampunk, etc), as it'll get confusing when Embedded Steampunk starts supporting OAuth (I expect it to be slightly different to the CF OAuth flow that we have right now).

Reply:
Looks good